.gitignore

```diff
@@ -9,4 +9,3 @@ install_manifest.txt
 compile_commands.json
 CTestTestfile.cmake
 OSP-MAGNUM.cbp
-
```

Comment:
Please don't modify the file if you aren't making changes to it.

My advice is to use "git diff" prior to submitting the code just as a sanity check to avoid unexpected changes.